spec/system/admin/admin_manages_pages_spec.rb

```diff
@@ -111,4 +112,37 @@
       end
     end
   end
+
+  describe "when there is a landing page" do
+    let(:current_landing_page) { create(:page, landing_page: true) }
+    let(:new_landing_page) { create(:page, landing_page: true) }
+
+    it "allows a landing page to be updated", :aggregate_failures do
+      visit edit_admin_page_path(current_landing_page.id)
+      expect(page).to have_content("Use as 'Locked Screen")
+      uncheck "Use as 'Locked Screen'"
+      click_on("Update Page")
+      expect(page).to have_current_path(admin_pages_path)
+    end
+
+    it "allows an Admin to click through to the current landing page via the modal", :aggregate_failures do
+      visit edit_admin_page_path(new_landing_page.id)
+      expect(page).to have_content("Use as 'Locked Screen")
+      check "Use as 'Locked Screen'"
+      expect(page).to have_link("Current Locked Screen")
+      click_on("Current Locked Screen")
+      expect(page).to have_current_path(new_landing_page.path)
+      expect(page).to have_content(new_landing_page.title)
+    end
+
+    xit "allows an Admin to overwrite the current landing page via the checkbox and modal", :aggregate_failures do
+      visit edit_admin_page_path(new_landing_page.id)
+      expect(page).to have_content("Use as 'Locked Screen")
+      check "Use as 'Locked Screen'"
+      expect(page).to have_link("Current Locked Screen")
+      click_on("Overwrite current locked screen")
+      click_on("Update Page")
+      expect(page).to have_current_path(admin_pages_path)
```

Comment:
I commented this test out since I've been unable to get it to pass due to an ambiguous element, the "x" in the top right corner of the modal. In order to properly overwrite a landing page, an Admin has to click the "Over current locked screen" button and _then_ exit out of the modal and click the "Update Page" button to complete the overwriting. This said, I've tried a handful of different approaches to target the "x" to no avail. Below are some of the different ways I've tried to target the ambiguous element:
1. `click_button(class: "crayons-btn crayons-btn--ghost crayons-btn--icon")`
2. `find(:css, "svg.crayons-icon").click`
3. `find(:xpath, "//*[@id='add-landing-page-link-modal-root']/div/div[1]/div[1]/button").click`
4. I've also tried targeting the element by nesting the clicks within `within` blocks to help with specificity as shown below:
```
within("#crayons-modal__box") do
   click_button(class: "crayons-btn crayons-btn--ghost crayons-btn--icon")
end
```
```
within("crayons-btn crayons-btn--ghost crayons-btn--icon") do
  find(:css, "svg.crayons-icon").click
end
```

Any ideas on how to properly click this element would be so greatly appreciated!!

Reply:
Thanks so much for the pointer, @aitchiss!